docker/Dockerfile

```diff
@@ -82,15 +82,47 @@ RUN echo "set -e" >> $WORKSPACE/.install-dependencies.sh && \
     rm -rf /var/lib/apt/lists/*
 
 # add additionally specified apt dependencies to install script
+# TODO: if repository is top-level package, WORKDIR would need to be $WORKSPACE/src/target/<PACKAGE>
```

Comment:
TODO: if repository is top-level package, WORKDIR would need to be $WORKSPACE/src/target/<PACKAGE>